server/service/src/backend_plugin/types/transform_requisition_lines.rs

```diff
@@ -15,6 +15,7 @@ pub struct Input {
 #[ts(rename = "TransformRequisitionLineOutput")]
 pub struct Output {
     pub transformed_lines: Vec<RequisitionLineRow>,
+    pub plugin_data: Option<Vec<PluginDataRow>>,
```

Comment:
Plugin data is now returned by plugin, didn't want to save it during plugin execution, to still have transaction rollback